### PR TITLE
Adding cumulative reporting of stats for a time window

### PR DESCRIPTION
**Pull request by rahulmishra0802**

### CODE REVIEW

File: cogniac/edgeflow.py
Comment (wskish) [inline cogniac/edgeflow.py:355]: probably should use start_time and end_time here to make this more closely match the API.  can default the values to last five minutes if not specified by user

**Review by wskish — CHANGES_REQUESTED** [review]

File: cogniac/edgeflow.py
Comment (spatel-cog) [inline cogniac/edgeflow.py:371]: this is the app_id not tenant_id

File: cogniac/edgeflow.py
Comment (spatel-cog) [inline cogniac/edgeflow.py:372]: can you use a different variable name here rather than status as it's already used a place holder for the generator at the start of the for-loop

File: cogniac/edgeflow.py
Comment (spatel-cog) [inline cogniac/edgeflow.py:382]: seconds might be better here.

File: cogniac/edgeflow.py
Comment (spatel-cog) [inline cogniac/edgeflow.py:384]: besides the total tput, do you want to provide a breakdown on a per-app basis?

File: cogniac/edgeflow.py
Comment (rahulmishra0802) [reply cogniac/edgeflow.py:355]: We can do that. But does someone actually provide time in the timestamp format? It's not very intuitive.

File: cogniac/edgeflow.py
Comment (rahulmishra0802) [reply cogniac/edgeflow.py:371]: Will correct

File: cogniac/edgeflow.py
Comment (rahulmishra0802) [reply cogniac/edgeflow.py:372]: sure

File: cogniac/edgeflow.py
Comment (rahulmishra0802) [reply cogniac/edgeflow.py:382]: Agreed. Seconds would be better

File: cogniac/edgeflow.py
Comment (rahulmishra0802) [reply cogniac/edgeflow.py:384]: Sure, should we go for per-app basis and total tput OR just the per-app basis? 
@spatel-cog @wskish

Comment (spatel-cog) [reply cogniac/edgeflow.py:384]: I think per-app and total tput for during this measurement interval would be useful - the replicas for a given app would have to be accounted for.

Comment (rahulmishra0802) [reply cogniac/edgeflow.py:384]: I think replicas would only be reflected in GPU_IDs. So code would handle that fine.

File: cogniac/edgeflow.py
Comment (wskish) [inline cogniac/edgeflow.py:370]: this filters based on cloudtime timestamp, which can be delayed relative to actual edgeflow timestamp by some arbitrary time if the connection to cloudcore is slow or temporarily disconnected

File: cogniac/edgeflow.py
Comment (wskish) [inline cogniac/edgeflow.py:376]: add additional filter based on edgeflow timestamp

**Review by wskish — CHANGES_REQUESTED** [review]

File: cogniac/edgeflow.py
Comment (rahulmishra0802) [reply cogniac/edgeflow.py:370]: In that case, we might have to fetch all the event reports that has happened after the start timestamp on that EF. This will make the code super slow.

File: cogniac/edgeflow.py
Comment (rahulmishra0802) [reply cogniac/edgeflow.py:376]: sure, I will add his filter.

File: setup.py
Comment (rahulmishra0802) [inline setup.py:9]: Should I update the version? 
If yes, should it be `2.0.14`?

**Comment by rahulmishra0802** [comment]
I have created the PR for public-api changes. https://github.com/Cogniac/public-api/pull/3361 
We need to wait for it to be deployed before we merge this.

**Review by wskish — APPROVED** [review]

**Pull request merged**
